WordPress/gutenberg
# Perform a complete draft save on preview

closes #12047
fixes #9151

This is a follow-up to https://github.com/WordPress/gutenberg/pull/12037#issuecomment-439753179 and it builds on top of some changes in #11409 

This PR changes the behavior of the preview button. It now performs a full draft save (on non published posts) before previewing the post. This matches Core behavior. (classic editor)

**Testing instructions**

 - Create a post
 - Set a featured image
 - Click preview
 - You should see the post with the featured image
